### ISSUE TITLE
[BUG] PaLM 2 API not working

### ISSUE THREAD

**Issue by ChessScholar** [SillyTavern/SillyTavern#1386]
### Environment

Self-Hosted (Bare Metal)

### System

Windows 10

### Version

1.10.9

### Desktop Information

node v20.9.0

### Describe the problem

I was able to have PaLM API working a few days ago, but it is now suddenly throwing me an error and cannot connect.  I haven't done anything new. I have already fully uninstalled and reinstalled everything.

![Screenshot 2023-11-22 094247](https://github.com/SillyTavern/SillyTavern/assets/65353254/d90f21b6-8899-41b4-9530-0937e4258dbf)

I have tested the PaLM API in a clean document and it works just fine. Even the Window AI of PaLM works.

I also looked into the PaLM API, and it doesn't look like I hit a max-quota mark either.

Please let me know what other information you need. Thanks for the help.

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You've checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by ChessScholar** [comment]
I figured it out how to get the API working, though I am not sure if this is the correct way to solve the issue.

In the SillyTavern\src\constants.js file, I simply removed the following line:
`    {
        category: "HARM_CATEGORY_UNSPECIFIED",
        threshold: "BLOCK_NONE"
    },`
    
I spent about 5 hours last night researching trying to figure it out. I guess sleeping on it helped. 

I'll close this issue once confirmed by admins that this is the correct solution. Thanks.
